tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java

```diff
@@ -24,21 +24,33 @@
 
 package test.robot.javafx.scene.layout;
 
-import test.robot.javafx.scene.layout.RegionUITestBase;
-import javafx.geometry.Bounds;
-import javafx.scene.paint.Color;
+import static org.junit.Assume.assumeTrue;
+
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
+
 import com.sun.javafx.PlatformUtil;
-import static org.junit.Assume.assumeTrue;
 
+import javafx.geometry.Bounds;
+import javafx.scene.paint.Color;
+
+/**************************************************************************
+ *                                                                        *
+ * Tests for aligned background images. The test image in use is chosen   *
+ * to align naturally with the edge of the test region.                   *
+ *                                                                        *
+ *************************************************************************/
 public class RegionBackgroundImageUITest extends RegionUITestBase {
-    /**************************************************************************
-     *                                                                        *
-     * Tests for aligned background images. The test image in use is chosen   *
-     * to align naturally with the edge of the test region.                   *
-     *                                                                        *
-     *************************************************************************/
+
+    @Before
+    public void before() {
```

Comment:
You might consider moving this method to the `VisualBase` superclass, since it would be possible for any other robot-based screen capture test to fail in the future. If you do that, you wouldn't have a stage that you could access, so you would need to do something like move the mouse to the lower right of the screen rather than the window.